src/sentry/testutils/fixtures.py

```diff
@@ -466,6 +466,7 @@ def create_useremail(self, user, email, **kwargs):
         return useremail
 
     def create_event(self, event_id=None, normalize=True, **kwargs):
+        # XXX: Do not use this method for new tests! Prefer `store_event`.
```

Comment:
Can we emit a warning to stdout/stderr if this is being called directly from a test?

Reply:
Most tests still use this. I am not sure how to fix this situation. I would propose requiring a kwarg `is_legacy_test=True` to deter from using this in new tests.

Migrating the entire testsuite is unlikely to ever happen IMO.